invenio_app_ils/records/serializers/__init__.py

```diff
@@ -55,3 +57,12 @@
 
 loan_v1_search = search_responsify(loan_v1, 'application/json')
 """JSON response builder decorates loan."""
+
+literature_v1 = LiteratureJSONSerializer(ILSRecordSchemaJSONV1, replace_refs=True)
```

Comment:
Please move this in the `literature` module, check how it has been done in [my PR](https://github.com/inveniosoftware/invenio-app-ils/pull/820) or rebase from there since it will be merged soon.

Reply:
I left it here on purpose because was expecting your pr to be merged first and then rebase it on top. Will move it in literature as you suggested.